testutils/block.go

```diff
@@ -0,0 +1,46 @@
+package testutils
+
+import (
+	"github.com/renproject/hyperdrive/block"
+	"github.com/renproject/hyperdrive/sig"
+)
+
+func SignBlock(blk block.Block, signer sig.SignerVerifier) *block.SignedBlock {
+	signedBlock, _ := blk.Sign(signer)
```

Comment:
Never ignore the error. Even in a test suite this should be, at the very least, a panic. Otherwise how would you ever know if this error had occurred?